devsetup/scripts/tripleo.sh

```diff
@@ -39,43 +41,88 @@ if [[ ! -f $SSH_KEY_FILE ]]; then
     exit 1
 fi
 
+sudo dnf install -y python-jinja2
 source ${SCRIPTPATH}/common.sh
 
+# Clock synchronization is important for both Ceph and OpenStack services, so both ceph deploy and tripleo deploy commands will make use of chrony to ensure the clock is properly in sync.
+# We'll use the NTP_SERVER environmental variable to define the NTP server to use, e.g.:
+# export NTP_SERVER=pool.ntp.org
+
+if [ $EDPM_COMPUTE_CELLS -eq 2 ] || [ $EDPM_COMPUTE_CELLS -gt 3 ] || [ $EDPM_COMPUTE_CELLS -eq 0 ] ; then
```

Comment:
can we instead explicitly search for what IS allowed ? i.e. the "main cell0 plus additional 2 compute cells"

Reply:
do you mean state in the echo message? what are we going to "search" here?

Reply:
like if EDPM_COMPUTE_CELLS != 3 (or whatever the allowed is i think 3?) then error

Reply:
This already explicitely checks for allowed values of a 1 and 3 :)

Reply:
`if [ $EDPM_COMPUTE_CELLS -ne 1 -a $EDPM_COMPUTE_CELLS -ne 3 ]; then ...`